src/wiki/core/markdown/mdx/responsivetable.py

```diff
@@ -7,7 +7,10 @@ class ResponsiveTableExtension(markdown.Extension):
     """Wraps all tables with Bootstrap's table-responsive class"""
 
     def extendMarkdown(self, md):
-        md.treeprocessors.add("responsivetable", ResponsiveTableTree(md), "_end")
+
+        md.treeprocessors._sort()
+        priority = md.treeprocessors._priority[-1].priority - 5
```

Comment:
This looks like the best solution, given the options. It's a pity that wanting to put an extension at the end isn't a very clearly defined argument.